guidelines/licenses.md

```diff
@@ -0,0 +1,9 @@
+# License
+
+You can select license during repository creation. If you forgot doing that then follow this [instruction](https://help.github.com/en/articles/adding-a-license-to-a-repository).
+
+If `year` is required use current year value.
```

Comment:
http://updateyourfooter.com/ 😉 
Maybe we should automate that.

Reply:
not sure how in md in repo